### PR TITLE
Yubico

### PR DESCRIPTION
**Pull request by stintel**
Maintainer: @stuwilkins 
Compile tested: n/a
Run tested: n/a

Description:
Fix typo and refresh patches while at it to make CI happy.

### CODE REVIEW

**Review by aparcar — APPROVED** [review]

File: libs/libyubikey/Makefile
Comment (aparcar) [inline libs/libyubikey/Makefile:5]: Consider using `$(AUTORELEASE)` if you feel fancy.

Comment (stintel) [reply libs/libyubikey/Makefile:5]: I don't. Let's leave that up to the maintainer?

**Review by neheb — APPROVED** [review]

**Pull request merged**
